### ISSUE TITLE
Link to Image

### ISSUE THREAD

**Issue by florian4300** [htl-perg-2019-20-5ahif/docker-wpf-exercise-florian4300#1]
@rstropek 
I finished the example and here is the link to the image on DockerHub:
https://hub.docker.com/repository/docker/florian4300/csharp_carbooking.
I also created a yaml file, which sets up the API + SQL Server automatically. 
The WPF can be used too.

**Comment by rstropek** [comment]
@florian4300 Very well done 👍! I was looking for the Dockerfile for your API but couldn't find it. Did you check it in?

Greetings,
Rainer Stropek.

**Comment by florian4300** [comment]
@rstropek 
Yes. Sorry I frogot to push my last commit. There were also some bugfixes and changes to the WPF App.
